crates/rpc-types-engine/src/jwt.rs

```diff
@@ -160,7 +160,7 @@ impl Default for Claims {
 /// for the JWT, which is included in the JWT along with its payload.
 ///
 /// See also: [Secret key - Engine API specs](https://github.com/ethereum/execution-apis/blob/main/src/engine/authentication.md#key-distribution)
-#[derive(Copy, Clone, PartialEq, Eq)]
+#[derive(Copy, Clone, PartialEq, Eq, Default)]
```

Comment:
One potential concern with defaulting a secret value is that it could make it easier for users to forget to fill it as it falls through

Reply:
Yes I agree, as you prefer, feel free to merge it or not... I think rn could be a blocker for deriving Default in some objects using this, like I mention in the pr description. But I understand your concern.